### DISCUSSION TITLE
Ian Wold | I've Stopped Using Visual Studio

### DISCUSSION THREAD

**Discussion by giscus** [IanWold/ianwold.github.io#16] [Posts]
# Ian Wold | I've Stopped Using Visual Studio

Ian Wold is a Software Engineer, Architect, and Team Leader in Minneapolis.

https://ian.wold.guru/Posts/ive_stopped_using_visual_studio.html

**Comment by andrewwillette** [comment]
Nice! maybe the vim plugin too ;)
